lib/whitespace.coffee

```diff
@@ -88,3 +93,19 @@ class Whitespace
 
     buffer.transact ->
       buffer.scan new RegExp(spacesText, 'g'), ({replace}) -> replace('\t')
+
+  # returns null if no current Git-repo or file is new and unsaved
+  # returns Array[Number] where each index is a modified row
+  _getModifiedRows: (editor) ->
```

Comment:
We don't use underscores to mark methods private in Atom. We just either don't put a doc comment in or mark it private like so:

```coffee
# Private: Gets the list of modified rows.
#
# Returns null if there is no repository or file is new and unsaved.
# Returns {Array} of modified row {Number}.
getModifiedRows: (editor) ->
```